libs/pages/services/src/lib/ui/page-settings/page-settings.tsx

```diff
@@ -10,11 +10,11 @@ export function PageSettings(props: PageSettingsProps) {
   const { links, children } = props
 
   return (
-    <div className="bg-white flex mt-2 min-h-[calc(100%-200px)] rounded-sm">
-      <div className="w-72 shrink-0 pt-6 border-r border-element-light-lighter-400">
-        <NavigationLeft links={links} />
+    <div className="bg-white flex mt-2 rounded-sm">
+      <div className="w-72 border-r border-element-light-lighter-400 relative min-h-[calc(100vh-270px)] pb-10">
```

Comment:
Is not that a min-h value that we should put in a theme ? I guess we are gonna use it in many different places and it's nice that we have only to edit the theme to have the changes everywhere no?

Reply:
Yep done I added min-h !